Description: Allow Routes == 2.0
 The Debian package for Routes 2.0 is perfectly valid, as it contains patches.
 This patch makes it possible to use it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-08-02

--- heat-5.0.0~b2.orig/requirements.txt
+++ heat-5.0.0~b2/requirements.txt
@@ -49,8 +49,7 @@ python-troveclient>=1.0.9
 python-zaqarclient>=0.1.1
 PyYAML>=3.1.0
 requests>=2.5.2
-Routes!=2.0,!=2.1,>=1.12.3;python_version=='2.7'
-Routes!=2.0,>=1.12.3;python_version!='2.7'
+Routes>=1.12.3
 six>=1.9.0
 SQLAlchemy<1.1.0,>=0.9.7
 sqlalchemy-migrate>=0.9.6
